ERROR: failed to solve: DeadlineExceeded: context deadline exceeded

Sorry for the vague error message.

We have a docker image that runs some things. Most times, it runs fine. Occasionally, it errors at the very end with a vague error message that is not from the yml file but I think is from the docker image exiting/ending. (All our commands run fine in the container. It is failing after the last one completes).

We run the image from a github action, if that matter.

The question is:

* What is not being solved?
* What deadline is exceeded?
* Can it be increased?
* Is there some way of running the container in debug mode for more information?

Thanks in advanced.

```
2023-10-04T20:39:13.3582748Z #24 exporting to client directory
2023-10-04T20:39:14.5610603Z #24 copying files 28B
2023-10-04T20:39:19.7302573Z #24 copying files 1.75GB 5.2s
2023-10-04T20:39:24.6910750Z #24 copying files 3.81GB 10.2s
2023-10-04T20:39:29.7983597Z #24 copying files 5.89GB 15.3s
2023-10-04T20:39:35.6174985Z #24 copying files 7.41GB 21.1s
2023-10-04T20:39:49.8824772Z #24 copying files 7.69GB 35.4s
2023-10-04T20:39:54.8880765Z #24 copying files 10.02GB 40.4s
2023-10-04T20:40:04.0740589Z #24 copying files 10.52GB 49.4s done
2023-10-04T20:40:04.0741210Z #24 DONE 50.7s
2023-10-04T20:41:33.3139725Z ERROR: failed to solve: DeadlineExceeded: context deadline exceeded
2023-10-04T20:41:33.3615053Z ##[error]Process completed with exit code 1.
2023-10-04T20:41:33.4182323Z Cleaning up orphan processes
```
```
[epasveer@ws7-01 ~]$ docker --version
Docker version 20.10.8, build 3967b7d
[epasveer@ws7-01 ~]$
```

--- a/solver/llbsolver/solver.go
+++ b/solver/llbsolver/solver.go
@@ -198,7 +198,7 @@ func (s *Solver) recordBuildHistory(ctx context.Context, id string, req frontend
 		}
 
 		ctx, cancel := context.WithCancelCause(context.Background())
-		ctx, _ = context.WithTimeoutCause(ctx, 20*time.Second, errors.WithStack(context.DeadlineExceeded))
+		ctx, _ = context.WithTimeoutCause(ctx, 300*time.Second, errors.WithStack(context.DeadlineExceeded))
 		defer cancel(errors.WithStack(context.Canceled))
 
 		var mu sync.Mutex

